backends-common/cassandra/src/test/resources/logback-test.xml

```diff
@@ -1,28 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-
-        <contextListener class="ch.qos.logback.classic.jul.LevelChangePropagator">
-                <resetJUL>true</resetJUL>
-        </contextListener>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
 
-        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-                <encoder>
-                        <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
-                        <immediateFlush>false</immediateFlush>
-                </encoder>
-        </appender>
+    http://www.apache.org/licenses/LICENSE-2.0
 
-        <root level="WARN">
-                <appender-ref ref="CONSOLE" />
-        </root>
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<configuration>
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
 
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
+            <immediateFlush>false</immediateFlush>
+        </encoder>
+    </appender>
 
-        <logger name="org.apache.james" level="WARN" >
-                <appender-ref ref="CONSOLE" />
-        </logger>
+    <root level="ERROR">
```

Comment:
I don't agree to have ERROR level for tests, we clearly need as many logs as possible because most of the time, it saves us some painful reproduction on our local computer.
The problem IMO is not that the log level is too low but that we log on the console.
What about logging in files instead ?